regenerate.sh

```diff
@@ -76,4 +93,9 @@ rm ${WORKDIR}/out/google.golang.org/grpc/reflection/grpc_testingv3/*.pb.go
 # grpc/service_config/service_config.proto does not have a go_package option.
 cp ${WORKDIR}/out/grpc/service_config/service_config.pb.go internal/proto/grpc_service_config
 
+# grpc/credentials/tls/certprovider/meshca/config.proto does not have a
```

Comment:
Should we add a `go_package`?
It will become a problem if it's imported by another proto.

Reply:
But the source of truth will eventually be the istio repo. I'm adding our own proto here for the time being and so that I can be unblocked.